packages/circuits/circom/eddsa-proof.circom

```diff
@@ -0,0 +1,26 @@
+pragma circom 2.1.5;
+
+include "babyjub.circom";
+include "poseidon.circom";
+
+// This circuit can be used to prove the possession of a private key of an
+// identity commitment without revealing the private key itself. It utilizes 
+// the {@link https://eips.ethereum.org/EIPS/eip-2494|Baby Jubjub} elliptic curve, the Poseidon hash function which a highly efficient 
+// and secure hash function suited for zero-knowledge proof contexts.
+// A scope value can be used to define a nullifier to prevent the same
+// proof from being re-used twice.
+template EddsaProof() {
+    // The circuit takes two inputs: the private key and an additional scope parameter.
+    signal input secret;
+    signal input scope;
```

Comment:
It appears that the `scope` is not being used in the circuit. If this is the case, it can be removed. @cedoor @vplasencia 

If that's the case, we should adapt the tests accordingly and `eddsa-proof` package as well.

Reply:
I think it should stay there to allow devs to use it to prevent double-signaling. As we've discussed, nullifier = scope + commitment.

It needs a dummy square, like the `message` signal in the Semaphore circuit -> https://github.com/semaphore-protocol/semaphore/blob/596e10c210e5e94cb2b2d79089d1add92d52d1a2/packages/circuits/semaphore.circom#L24

Reply:
Makes sense and thank you for the explanation. Yes, we can just add the dummy square and that's it.

Reply:
Yess, I've opened another issue for `poseidon-proof`: https://github.com/privacy-scaling-explorations/zk-kit/issues/136